src/ansys/dpf/post/dataframe.py

```diff
@@ -0,0 +1,357 @@
+"""Module containing the ``DataFrame`` class."""
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, List, TypeVar, Union
+
+from ansys.dpf.core import FieldsContainer, ScopingsContainer
+from ansys.dpf.core.operators.utility import merge_scopings
+import numpy
+import numpy as np
+
+from ansys.dpf.post.errors import PandasImportError
+
+PANDAS_PRESENT = False
+if TYPE_CHECKING:
+    try:
+        import pandas as pd
+
+        PANDAS_PRESENT = True
+    except ImportError:
+        pass
+
+if PANDAS_PRESENT:
+    PandasDataFrameType = pd.DataFrame
+else:
+    PandasDataFrameType = TypeVar("PandasDataFrameType")
+
+display_width = 80
+display_max_colwidth = 16
```

Comment:
If you rename those global variables, you might have to rename them below -just to not forget about it 😃- .
```suggestion
DISPLAY_WIDTH = 80
DISPLAY_MAX_COLWIDTH = 16
```